Added Intros for REST API Pages (#8371)
github/docs: content/rest/reference/gitignore.md

---
title: Gitignore

> intro: 'The Gitignore API fethes all the available [.gitignore templates](https://github.com/github/gitignore) that can be used to ignore the dependencies from adding in your project's repositories.' 

Reviewer: Since the intro includes an apostrophe, you need to use double quotes instead of single quotes. Also typo fix:
```suggestion
intro: "The Gitignore API fetches all the available [.gitignore templates](https://github.com/github/gitignore) that can be used to ignore the dependencies from adding in your project's repositories."
```

> intro: "The Gitignore API fetches all the available [.gitignore templates](https://github.com/github/gitignore) that can be used to ignore the dependencies from adding in your project's repositories."

Reviewer: It would be more accurate to say that .gitignore is used to ignore files. Also, since the repo is linked in the next paragraph with more details, I would recommend omitting the link here.
```suggestion
intro: "The Gitignore API fetches `.gitignore` templates that can be used to ignore files and directories."
```
